test/trace_api_test.c

```diff
@@ -17,48 +17,67 @@ static int test_trace_categories(void)
 
     for (cat_num = -1; cat_num <= OSSL_TRACE_CATEGORY_NUM + 1; ++cat_num) {
         const char *cat_name = OSSL_trace_get_category_name(cat_num);
-        int is_cat_name_eq = 0;
+        const char *expected_cat_name = NULL;
         int ret_cat_num;
-        int expected_ret;
 
+#define SET_EXPECTED_CAT_NAME(name) expected_cat_name = #name; break
         switch (cat_num) {
-#define CASE(name) \
```

Comment:
I really don't get why there is a refactoring in here ... it seems totally unrelated to the rest of the PR

Reply:
It was the result of coding style errors, which I just happened to fix up here